src/BloomExe/XmlHtmlConverter.cs

```diff
@@ -10,25 +10,23 @@
 
 namespace Bloom
 {
-	public class XmlHtmlConverter
+	public static class XmlHtmlConverter
 	{
-		/// <summary>
-		///
-		/// </summary>
-		/// <param name="content"></param>
-		/// <exception cref="">Throws if there are parsing errors</exception>
-		/// <returns></returns>
+
+		private static readonly Regex _selfClosingRegex = new Regex(@"<([ubi]|span)(\s+[^><]+\s*)/>");
+		private static readonly Regex _emptySelfClosingRegex = new Regex(@"<([ubi]|span)\s*/>");
+		private static readonly Regex _emptyTagsRegex = new Regex(@"<([ubi]|span)([^>]*?)>(\s*)</\1>");
+
```

Comment:
Also missing the point of \s*. You're already matching anything that isn't >, won't that include white space?